geotrek/tourism/tests/test_parsers.py

```diff
@@ -1053,3 +1053,85 @@ def mocked_download(*args, **kwargs):
         self.assertAlmostEqual(information_desk.geom.y, 6276967.321705549, places=5)
         self.assertEqual(str(information_desk.photo), '')
         self.assertEqual(InformationDesk.objects.exclude(photo='').first().photo.read(), b'boo')
+
+
+class TestInformationDeskOpenStreetMapParser(InformationDeskOpenStreetMapParser):
+    type = "Foo"
+    tags = {"amenity": "ranger_station"}
+
+    def filter_name(self, src, val):
+        if val:
+            return val
+        else:
+            return "test_flexible"
+
+
+class OpenStreetMapParserTests(TestCase):
+    @mock.patch('geotrek.common.parsers.requests.get')
+    def import_information_desk(self, mocked):
+        def mocked_json():
+            filename = os.path.join(os.path.dirname(__file__), 'data', 'information_desk_OSM.json')
+            with open(filename, 'r') as f:
+                return json.load(f)
+
+        mocked.return_value.status_code = 200
+        mocked.return_value.json = mocked_json
+
+        call_command('import', 'geotrek.tourism.tests.test_parsers.TestInformationDeskOpenStreetMapParser')
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.type = InformationDeskTypeFactory.create(label="Foo")
+
+        # None argument is for mocked positional argument
+        cls.import_information_desk(None)
```

Comment:
I think itg's not a best practice to replace a mock aerg with None. I sugest you in this case to define your mock at Class level instead method, and don't use  None as param